File: new/atoms/support-chats.test.mjs
import * as m from './support-chats.mjs';
const SNAP = {"SUPPORT_CHATS":"\"supportChats\""};
let f = 0;
for (const [k, s] of Object.entries(SNAP)) if (JSON.stringify(m[k]) !== s) { console.error('✗ ' + k + ' סטה מהצילום'); f = 1; }
if (f) process.exit(1); console.log('✓ support-chats: צילום-ערך תואם — ירוק');

File: new/dart-maor/support-chats_test.dart
// בדיקת-חוזה (רתמת-זהב) · support-chats — צילום-ערך jsonEncode≡JSON.stringify.
// חוללה מכנית (gen-data-dart). הרצה: dart run --enable-asserts ⇒ OK
import 'dart:convert';
import 'support-chats.dart';

void main() {
  // ‏1) צילום-ערך: jsonEncode ≡ JSON.stringify של המקור (זהה-ביט).
  if (jsonEncode(supportChats) != '"supportChats"') {
    throw StateError('FAIL supportChats: צילום-הערך סטה');
  }
  print('OK support-chats: 1 data-snapshot(s) passed');
}
